sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -201,14 +215,19 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
 
   private fun checkSurfaceSizeChanged() {
     if (sizeChanged) {
-      mapboxRenderer.onSurfaceChanged(
-        width = width,
-        height = height
-      )
+      mapboxRenderer.onSurfaceChanged(width = width, height = height)
+      widgetRenderer.onSurfaceChanged(width = width, height = height)
       sizeChanged = false
     }
   }
 
+  private fun checkWidgetRender() {
+    if (eglPrepared && !widgetRenderCreated && widgetRenderer.hasWidgets()) {
+      widgetRenderer.onSharedContext(eglCore.eglContext)
```

Comment:
doubt a bit about naming... I guess it's mostly something equivalent to `prepareEgl` method we have, perhaps go with `prepareWithSharedContext`? naming with `on` is more of a callback but here it's more of an action

Reply:
discussed offline, closing